diesel/src/macros/associations/belongs_to.rs

```diff
@@ -1,5 +1,7 @@
 #[macro_export]
 #[doc(hidden)]
+#[cfg(feature = "with-deprecated")]
+#[deprecated(since = "1.1.0", note = "Use `#[derive(Associations)]` instead")]
```

Comment:
This isn't public API. We don't need to deprecate it.

Reply:
Technically this is public API, because a downstream crate could use this. Removing this macro will cause those (theoretical existing) crates to break.

Reply:
Our definition of public API is whether or not it appears on documentation. `#[doc(hidden)]` items can be freely changed without deprecation.